tests/unit/test_dispatch.py

```diff
@@ -0,0 +1,44 @@
+#
+# Test dispatching mechanism in entry points
+#
+import pybamm
+
+
+class TestDispatch:
```

Comment:
Could this be parametrised across `pybamm.dispatch.models.get_entries("pybamm_models")`?